tests/tests_API_FB.py

```diff
@@ -9,21 +9,61 @@ def test_api_urls():
     """The testing will check if our functions return the required output"""
     # Create soup object & print a status message - 404 not good , 200 good
     soup = API_FB.create_soup(API_FB.create_req())
-    expected_output_urls = ['https://www.facebook.com/careers/v2/jobs/1672813472870915/',
-                            'https://www.facebook.com/careers/v2/jobs/712878282607325/',
-                            'https://www.facebook.com/careers/v2/jobs/420212419368641/',
-                            'https://www.facebook.com/careers/v2/jobs/207984144060781/',
-                            'https://www.facebook.com/careers/v2/jobs/773785633436218/',
-                            'https://www.facebook.com/careers/v2/jobs/2597607417128369/',
-                            'https://www.facebook.com/careers/v2/jobs/536078547072736/']
+    expected_output_urls = ['https://www.facebook.com/careers/v2/jobs/1035151026857199/',
```

Comment:
Are you sure that they will always be returned in the exact same order?

Reply:
Yep, untill Facebook employee will change the list :)
Still looking for a solution for this